completions/bash/podman

```diff
@@ -1733,6 +1733,83 @@ _podman_logs() {
     esac
 }
 
+_podman_manifest() {
+	local boolean_options="
+    --help
+    -h
+    "
+    subcommands="
+     add
+	 create
+     inspect
+    "
+     __podman_subcommands "$subcommands" && return
+
+     case "$cur" in
+    -*)
+        COMPREPLY=( $( compgen -W "--help" -- "$cur" ) )
+        ;;
+    *)
+        COMPREPLY=( $( compgen -W "$subcommands" -- "$cur" ) )
+        ;;
+     esac
+}
+
+_podman_manifest_add() {
+	local options_with_args="
+      --annotation
+      --arch
+	  --features
+	  --os-version
+	  --variant
```

Comment:
Did you meant to have an extra couple of spaces before the last three lines here?